src/main/java/Duke.java

```diff
@@ -1,10 +1,35 @@
+import duke.*;
```

Comment:
Perhaps can split to multiple imports of duke classes?

Reply:
Sure. Thank you!